### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by ionelmc**

### CODE REVIEW

**Comment by ionelmc** [comment]
I don't get why it includes the commit from the previous pr.

**Comment by dhermes** [comment]
> I don't get why it includes the commit from the previous pr.

I did `merge --squash` (in the UI). You can rebase your branch if you like.

File: pypy2/python.pypy.nuspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>python.pypy</id>
-    <version>7.3.1</version>
+    <version>7.3.1.20200825</version>
```
Comment (dhermes) [inline pypy2/python.pypy.nuspec:6]: This version is just aesthetic / doesn't correspond to a "real" PyPy version?

Comment (ionelmc) [reply pypy2/python.pypy.nuspec:6]: It's the same scheme as cpython versioning. Eg https://chocolatey.org/packages/python/3.8.5.20200721 - last part is a tag date.

**Review by dhermes — APPROVED** [review]
LGTM, not sure if we want to merge or see if we can update the existing version in place?

**Comment by ionelmc** [comment]
I already tried republishing the artifacts, they need new version for sure.

**Comment by ionelmc** [comment]
Eg: 
image

**Pull request merged**